_benchmark/workloads/vectorsearch.md

```diff
@@ -0,0 +1,572 @@
+| `hnsw_m `                                 | The `HNSW m` parameter.                                                                                                                         |
+| `query_k`                                 | The number of neighbors to return for the search. Only one of `query_k`, `query_max_distance`, and, `query_min_score` can be provided.          |
+| `query_max_distance `                     | The maximum distance to be returned for the vector search. Only one of `query_k`, `query_max_distance`, and, `query_min_score` can be provided. |
+| `query_min_score`                         | The minimum score to be returned for the vector search. Only one of `query_k`, `query_max_distance`, and, `query_min_score` can be provided.    |
+| `query_data_set_format`                   | The format of the vector dataset used for queries. Only one of `query_k`, `query_max_distance`, and, `query_min_score` can be provided.        |
+| `query_data_set_path`                     | The path to the vector dataset used for queries.                                                                                               |
+| `query_count`                             | The number of queries for the search operation.                                                                                                 |
+| `query_body`                              | The JSON properties that will be merged with search body                                                                                        |
+| `search_clients`                          | The number of clients to use for running queries.                                                                                               |
+| `repetitions`                             | THe number of repetitions until the dataset is exhausted. Default is `1`.                                                                      |
+| `target_throughput`                       | The target throughput for each query operation in requests per second. Default is `10`.                                                         |
+| `time_period`                             | The period of time dedicated for the run the benchmark test in seconds. Default is `900`.                                                       |
+
+
+
+## Test procedures
+
+The workload supports the following test procedures.
+
+### No train test
+
+The no train test procedure tests vector search indices that require no training. You can define the underlying configuration of the vector search algorithm (such as specific engine or space type) as method definitions. 
+
+### No train test (Index only)
+
+The no train test (Index only) procedure is used to index vector search indexes that require no training. It's particularly useful when you want to benchmark only the indexing operation.
```

Comment:
```suggestion
The no train test (index only) procedure is used to index vector search indexes that require no training. This can be particularly useful when you want to benchmark only the indexing operation.
```